lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -908,6 +911,34 @@ function dosomething_campaign_set_est_timezone($date, $add_four = FALSE, $end_of
 }
 
 /**
+ * Returns array of date times ofor the given campaign node
+ *
+ * @param Object or Array
+ *    vars array or node object to pull from
+ *
+ * @return array $timing
+ *    today - The current day
+ *    start_date - The day the campaign starts
+ *    end_date - The day the campaign ends
+ */
+function dosomething_campaign_get_goals_times($node) {
+  if (!isset($node)) {
+    return NULL;
+  }
+  date_default_timezone_set('EST');
+  $high_season = is_array($node) ? $node['field_high_season'] : $node->field_high_season[LANGUAGE_NONE];
```

Comment:
are you sure? I logged this and it was either LANGUAGE_NONE or just 0 (depends how you access it, node vs vars), and this is how it always was, see link below 

https://github.com/DoSomething/phoenix/blob/16bde548324a55db89de482a57ea85ba3ea3b04c/lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module#L765

Was this field recently made translatable?